### PR TITLE
feat: display specberus errors in more friendly way

### LINKED ISSUE #16: Display echidna errors
As we already fetch the results like:

https://labs.w3.org/echidna/api/status?id=870bc1a8-ed54-4db6-a5a7-60f1c98d7a8c

It would be cool if pulled out the specberus failures and displayed them in a user-friendly manner.

### PR DESCRIPTION
**Pull request by sidvishnoi**
Closes https://github.com/w3c/respec-w3c-auto-publish/issues/16
image
image
image

### CODE REVIEW

File: specberus/l10n-en_GB.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
+exports.messages = {
```
Comment (sidvishnoi) [inline specberus/l10n-en_GB.js:1]: This file is cloned from w3c/specberus repo using `specberus.sh` script.

File: specberus/rules.json [hunk 2]
```diff
@@ -0,0 +1,1943 @@
+{
```
Comment (sidvishnoi) [inline specberus/rules.json:1]: This file is also cloned from w3c/specberus repo using `specberus.sh` script.

File: specberus/messages.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+// @ts-check
+// Based on https://github.com/w3c/specberus/blob/master/lib/l10n.js
```
Comment (sidvishnoi) [inline specberus/messages.js:2]: Tries to make messages cleaner for CI env (less HTML) and simplifies some code.

**Review by marcoscaceres — APPROVED** [review]
Forgot you would have to endure the l10 lib from specberus 😹

**Pull request merged**